projects/novo-elements/src/elements/data-table/cell-headers/data-table-header-cell-filter-header.component.ts

```diff
@@ -0,0 +1,42 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, EventEmitter, Input, Output } from '@angular/core';
+import { NovoLabelService } from '../../../services/novo-label-service';
+import { Helpers } from '../../../utils';
+
+@Component({
+  selector: 'novo-data-table-cell-filter-header',
+  template: `
+    <div class="header">
+      <novo-label>{{ label || labels.filters }}</novo-label>
+      <novo-button
+        theme="dialogue"
+        color="negative"
+        size="small"
+        icon="times"
+        (click)="clearFilter.emit()"
+        *ngIf="hasFilter"
+        data-automation-id="novo-data-table-filter-clear">
+        {{ labels.clear }}
+      </novo-button>
+    </div>
+  `,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NovoDataTableCellFilterHeader {
+  @Input() label: any;
```

Comment:
could this be string instead of any? Might be safer to keep it as any in case its like an int or something

Reply:
yeah i wanted to keep it open so it can be used however people want, but really it should never be anything other than a string or an int. so that's a good call. made this a bit stricter.